easybuild/easyconfigs/f/FastQC/FastQC-0.10.1-Java-1.7.0_80.eb

```diff
@@ -17,22 +17,23 @@ description = """FastQC is a quality control application for high throughput seq
  application to review the results of several different QC checks, or create an HTML based
  report which can be integrated into a pipeline."""
 
-toolchain = {'name': 'goolf', 'version': '1.4.10'}
+toolchain = {'name': 'dummy', 'version': 'dummy'}
 
 sources = ['%(namelower)s_v%(version)s.zip']
 source_urls = ['http://www.bioinformatics.babraham.ac.uk/projects/%(namelower)s']
 
 java = 'Java'
 javaver = '1.7.0_80'
+versionsuffix = '-%s-%s' % (java, javaver)
+dependencies = [(java, javaver)]
 
-dependencies = [('Java', '1.7.0_80', '', True)]
-
-install_cmd = "cp -a FastQC/* %(installdir)s && chmod +x %(installdir)s/fastqc"
+postinstallcmds = ['chmod +x %(installdir)s/fastqc']
```

Comment:
@boegel - Rather than defining a custom `install_cmd` I'm using the default install action of a `PackedBinary` which is to copy all the contents of the unpacked binary to the `installdir`. All, that is required is to then do a simple post-install fix of executable permissions on the `fastqc` file.

So the result should be the same - are you seeing something different?